src/include/ipc/channel_map.h

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2019 Intel Corporation. All rights reserved.
+//
+// Author: Marcin Maka <marcin.maka@linux.intel.com>
+//         Slawomir Blauciak <slawomir.blauciak@linux.intel.com>
+
+#ifndef __SOF_AUDIO_CHANNEL_MAP_H__
```

Comment:
Needs to be aligned with the rest of the guards:
`#ifndef __IPC_CHANNEL_MAP_H__`